books/forms.py

```diff
@@ -5,4 +5,14 @@ class PostBookForm(forms.ModelForm):
     
     class Meta:
         model = Book
-        fields = ('title', 'author', 'cover')
\ No newline at end of file
+        fields = ('title', 'author', 'cover')
+
+    def __init__(self, *args, **kwargs):
+        super(PostBookForm, self).__init__(*args, **kwargs)
+        self.fields['cover'].required = False
```

Comment:
You can remove the need for this by defining `blank=True` in the definition of your model field.
[Have a look at the options here](https://docs.djangoproject.com/en/2.1/topics/db/models/#field-options)